AppCenter/AppCenter/Internals/Storage/MSStorageBindableType.h

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#import <Foundation/Foundation.h>
+#import <sqlite3.h>
```

Comment:
I don't like introducing this import in headers, let's avoid it to reduce dependencies